Modules/Microsoft365DSC/DSCResources/MSFT_TeamsVoiceRoute/MSFT_TeamsVoiceRoute.psm1

```diff
@@ -208,34 +209,27 @@ function Set-TargetResource
     $notFoundGatewayList = @()
     foreach ($gateway in $OnlinePstnGatewayList)
     {
-        if ( -not ($existingGateways -match $gateway))
+        if (-not ($existingGateways -match $gateway))
         {
             $notFoundGatewayList += $gateway
         }
     }
 
-    if ($notFoundUsageList)
+    if ($notFoundGatewayList)
     {
         $notFoundGateways = $notFoundGatewayList -join ','
-        throw "Please create the Voice Gateway object(s) ($notFoundGateways) using `"TeamsVoiceRoute`""
+        throw "Please create the Voice Gateway object(s) ($notFoundGateways) using cmdlet `"New-CsOnlinePSTNGateway`""
```

Comment:
Could you outline this change? Does the current recommendation not work?

Reply:
No it doesn't, TeamsVoiceRoute cannot create voice gateways. You either create it with that cmdlet or a new resource needs to be added to M365DSC that does that.

Reply:
Thanks for the info.